rust/stracciatella/src/logger.rs

```diff
@@ -165,7 +181,12 @@ impl Logger {
 
     /// Sets the global log level to a specific value
     pub fn set_level(level: LogLevel) {
-        GLOBAL_LOG_LEVEL.store(level.into(), Ordering::Relaxed);
+        RuntimeLevelFilter::set_global_log_level(level.into())
+    }
+
+    /// Sets the global log level to a specific value
```

Comment:
I know this came from me, but there seems to be a copy-paste-error in the comment...